tests/ui/test_ocs_odf_upgrade_ui.py

```diff
@@ -0,0 +1,84 @@
+from ocs_ci.ocs.ui.base_ui import logger
+from ocs_ci.ocs.ui.views import locators, ODF_OPERATOR
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocs_upgrade import OCSUpgrade
+from ocs_ci.ocs.ui.deployment_ui import DeploymentUI
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+from ocs_ci.ocs.resources.storage_cluster import ocs_install_verification
+from ocs_ci.framework.testlib import ManageTest, skipif_ocs_version, skipif_ocp_version
+
+
+@skipif_ocp_version("<4.9")
+@skipif_ocs_version("<4.8")
```

Comment:
I think that this is also not valid when upgrading between z-streams upgrade of 4.9.0 to 4.9.1 and for 4.9->4.10 upgrade.
So this will require some other logic to cover those skips. Which can be actually done as first lines in test to be able to get version from CSV and play with z-stream version as well